data/json/effects_on_condition/melee_eocs.json

```diff
@@ -909,6 +910,43 @@
       }
     ]
   },
+  {
+    "type": "effect_on_condition",
+    "id": "IS_UNARMED",
+    "global": false,
+    "condition": { "not": "u_can_drop_weapon" },
```

Comment:
Since gloves *can* be dropped, does this only function if you are using naked hands?

Reply:
In this case, it exclusively checks to see if the item in the player's hands is capable of being dropped, i'll have to add `"not":  { "u_has_wielded_with_weapon_category": "BIONIC_WEAPONS" }`  so it does fire when the player is using a bionic weapon

Reply:
It should work as long as you don't have anything in your hand. I'm going to add something so that is also excludes integrated bionic weapons